flask_resources/views/collections.py

```diff
@@ -28,10 +28,17 @@ def __init__(self, *args, **kwargs):
         # is defined by response_loader decorator
         self.request_loader = None
 
+    @property
+    def resource_method(self):
```

Comment:
normal: If there selection wasn't a decorator we could simply call the method with an appropriate argument value an avoid having to create a switch property...

Reply:
covered in above :)